src/screens/app/Components/SettingsButtonVertical/SettingsButtonVertical.js

```diff
@@ -0,0 +1,19 @@
+import {IconButton} from 'react-native-paper';
+import React from 'react';
+
+export default class SettingsButtonHorizontal extends React.Component {
+  state = {
+    iconSize: 30,
+    settingsButtonVerticalIcon: 'dots-vertical',
+  };
+  handleSettingsClick = () => {};
+  render() {
+    return (
+      <IconButton
+        icon={this.state.settingsButtonVerticalIcon}
+        size={this.state.iconSize}
+        onPress={this.handleSettingsClick}
```

Comment:
Why are we naming this SettingsButtonHorizontal but utilizing the settingsButtonVerticalIcon? Do we want consistency?

Reply:
This is a mistake. I am going to create just a custom settings button where we just add a prop, it's an idea from @denny999222

Reply:
Created SettingsButton component with unit tests